packages/istanbul-lib-report/README.md

```diff
@@ -3,4 +3,28 @@
 [![Greenkeeper badge](https://badges.greenkeeper.io/istanbuljs/istanbul-lib-report.svg)](https://greenkeeper.io/)
 [![Build Status](https://travis-ci.org/istanbuljs/istanbul-lib-report.svg?branch=master)](https://travis-ci.org/istanbuljs/istanbul-lib-report)
 
-Core reporting utilities for istanbul. Docs to follow.
+Core reporting utilities for istanbul.
+
+## Example usage
+
+```js
+const libReport = require('istanbul-lib-report');
+const reports = require('istanbul-reports');
+
+// create a context for report generation
+const context = libReport.createContext({
+  dir: 'report/output/dir',
+  defaultSummarizer: 'summarizer name e.g. nested/flat/pkg',
```

Comment:
I think this should be a valid value, and have a comment describing what it is, so that people have an idea what they can copy/paste and then extend as necessary. I've been working a lot with Gatsby recently and their docs do a good job of this.